site/content/operational_guide/namespace_configuration.md

```diff
@@ -115,9 +115,14 @@ Deleting a namespace is a simple as using the `DELETE` `/api/v1/services/m3db/na
 `curl -X DELETE <M3_COORDINATOR_IP_ADDRESS>:<CONFIGURED_PORT(default 7201)>/api/v1/services/m3db/namespace/<NAMESPACE_NAME>`
 
 Note that deleting a namespace will not have any effect on the M3DB nodes until they are all restarted.
+
 ### Modifying a Namespace
 
-There is currently no atomic namespace modification endpoint. Instead, you will need to delete a namespace and then add it back again with the same name, but modified settings. Review the individual namespace settings above to determine whether or not a given setting is safe to modify. For example, it is never safe to modify the blockSize of a namespace.
+There is currently no atomic namespace modification endpoint. Instead, you will need to delete a namespace and then add it back again with the same name, but modified settings. Review the individual namespace settings above to determine whether or not a given setting is safe to modify. 
+
+{{% notice warning %}}
+For example, it is never safe to modify the blockSize of a namespace.
```

Comment:
maybe we should phrase this as a list of warnings / unsafe items, with this being the first?

Reply:
@wesleyk Good idea, but I think I'll do that when we get to the rewrite, which is kind of in progress right now anyway